### PR TITLE
Overall edit of PyPDF-Core files. 

### PR DESCRIPTION
**Pull request by PipKat**
Edits to example files were limited to those necessary to address style issues discovered by Vale. All content will need to be reviewed again after completion of the PR. Note that some diagrams in examples were not being generated (locally at least), and lots of formatting changes are necessary to the examples.

### CODE REVIEW

**Comment by PipKat** [comment]
@PProfizi Lots of "Build and Test" checks failing.

**Comment by PProfizi** [comment]
> @PProfizi Lots of "Build and Test" checks failing.

Hi @PipKat, yes this is normal, we are in the process of releasing PyPDPF-Core 0.6.0 due to a milestone, which means we had to release higher versions of its dependencies, potentially breaking the pipelines. The release should be done within two hours and everything should be working again.

**Comment by PipKat** [comment]
@RobPasMue I don't know how these example files got into this PR. I swear it wasn't like this on Friday. It was just failing Build and Test checks.

**Comment by RobPasMue** [comment]
Hi @PipKat from what I can see, the files where already there. And by updating the examples they got also updated... Probably @PProfizi can tell us a bit more. In any case, the examples files shouldn't be in the ``docs/source/examples/*`` directory. They should be in the ``examples`` folder at the root of the repository. These are all auto-generated files.

**Comment by PProfizi** [comment]
@PipKat @RobPasMue I am not sure when this got here, apparently there might be a commit missing somewhere where they were removed, but I'll remove it.

**Comment by PipKat** [comment]
@PProfizi @RobPasMue  YES! All checks have passed. Now for a review?

**Review by PProfizi — CHANGES_REQUESTED** [review]
I think the docs/source/index.rst got mixed up as it is now made as if referring to PyDPF-Post.

File: docs/source/user_guide/waysofusing.rst [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
 whether you want to use standalone DPF or DPF in Mechanical before 
 creating any scripts. 
 
-``Standalone DPF`` uses CPython and can be accessed via any Python console. 
-Data can be exported to universal file formats (VTK, hdf5, txt files). 
-Use it to generate TH-plots, screenshots, animations, and so on, or create 
-custom results plots using numpy and matplotlib libraries.
+CPython
+-------
+Standalone DPF uses CPython and can be accessed via any Python console.
```
Comment (PipKat) [inline docs/source/user_guide/waysofusing.rst:13]:
```suggestion
Standalone DPF uses CPython and can be accessed with any Python console. 
```

File: docs/source/user_guide/waysofusing.rst [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
+files. You can use it to generate TH-plots, screenshots, and animations or
+to create custom result plots using `numpy <https://numpy.org/>`_
+and `matplotlib <https://matplotlib.org/>`_ packages.
 
 .. image:: ../images/drawings/dpf-reports.png
 
-``DPF in Mechanical`` uses IronPython and is accessible via the ACT console. 
+IronPython
+----------
+DPF in Mechanical uses IronPython and is accessible via the **ACT Console**.
```
Comment (PipKat) [inline docs/source/user_guide/waysofusing.rst:23]:
```suggestion
DPF in Mechanical uses IronPython and is accessible with the **ACT Console**. 
```

File: examples/07-python-operators/02-python_operators_with_dependencies.py [hunk 3]
```diff
@@ -133,19 +142,21 @@
...
+#   is located.
+# - The second argument is ``py_`` plus any name identifying the plug-in package.
+# - The third argument is the name of the function exposed in the ``__init__ file``
+#   for the plug-in package that is used to record operators.
 
 from ansys.dpf import core as dpf
 from ansys.dpf.core import examples
 
-# python plugins are not supported in process
+# Python plugins are not supported in process
```
Comment (PipKat) [inline examples/07-python-operators/02-python_operators_with_dependencies.py:159]:
```suggestion
# Python plugins are not supported in process.
```

**Comment by PipKat** [comment]
@PProfizi Hi, Paul. I'm sure I introduced those problems somehow when I pulled in changes from the main branch. I ended up with conflicts in these files. I'm not sure what I did wrong since it seems so long ago that I took that action, but thanks for catching the issues. I've incorporated your feedback to date and hope that you can find time to finish your review and approve the PR. Thanks!

**Comment by PipKat** [comment]
@PProfizi The Build and Test (3.10, ubuntu-latest check has been running the entire day!

**Review by PProfizi — APPROVED** [review]

**Pull request merged**